src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Test/packages.config

```diff
@@ -5,7 +5,8 @@
   <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.Resources" version="2.18.14-preview" targetFramework="net45" />
   <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.28.1" targetFramework="net45" />
-  <package id="Microsoft.Rest.ClientRuntime" version="2.3.2" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.3" targetFramework="net45" />
```

Comment:
The versions of Microsoft.Rest.ClientRuntime and Microsoft.Rest.ClientRuntime.Azure need to be the same across all modules.  Please standardize on the version currently in use.